internal/unix/types_other.go

```diff
@@ -21,6 +21,7 @@ const (
 	ESRCH  = syscall.ESRCH
 	ENODEV = syscall.ENODEV
 	EBADF  = syscall.Errno(0)
+	E2BIG  = syscall.Errno(0x7)
```

Comment:
Sorry for the confusion, we've recently decided to use 0 for future additions to non-Linux consts. This const doesn't exist on other platforms, so copying the value Linux uses might cause unintended side effects. (0x7 might be used by another const on a certain platform)